tests/lib/prepare-project.sh

```diff
@@ -32,6 +44,28 @@ build_deb(){
     cp ../*.deb "$GOHOME"
 }
 
+fedora_build_rpm() {
+    release=$(echo "$SPREAD_SYSTEM" | awk '{split($0,a,"-");print a[2]}')
+    arch=x86_64
+
+    base_version="$(head -1 debian/changelog | awk -F'[()]' '{print $2}')"
+    version="1337.$base_version"
```

Comment:
I am following the same scheme as the debian build does here. I guess 1337 is set to have a high enough version so that it overrides any other possibly installed versions. @mvo5 might know more. Just for the sake of the build and nothing which will ever end in the tree.